source/common/protobuf/utility.h

```diff
@@ -138,7 +138,7 @@ class ProtoValidationException : public EnvoyException {
 
 enum class ProtoUnknownFieldsMode { Strict, Allow };
 
-class MessageUtil {
+class MessageUtil : public Logger::Loggable<Logger::Id::config> {
```

Comment:
I agree with @alyssawilk, this is not in a config related file/directory, so would prefer not to log with this. Can we pass logger as a parameter, or maybe subclass `MessageUtil` in `config/` with a logger mixin?

Reply:
Or go back to MISC_LOG and claim there's precedent :-D

Reply:
Ok that's fair :)